nucypher/utilities/sandbox/blockchain.py

```diff
@@ -205,7 +206,7 @@ def time_travel(self, hours: int = None, seconds: int = None, periods: int = Non
 
     @classmethod
     def bootstrap_network(cls,
-                          economics: TokenEconomics = None
+                          economics: StandardTokenEconomics = None
```

Comment:
Same as above - `BaseEconomics` instead of `StandardTokenEconomics`

Reply:
:heavy_check_mark: